src/script/cpp_api/s_player.cpp

```diff
@@ -123,6 +124,20 @@ bool ScriptApiPlayer::on_prejoinplayer(
 	return false;
 }
 
+bool ScriptApiPlayer::can_bypass_userlimit(const std::string &name, const std::string &ip)
```

Comment:
I don't like it that `can_bypass_userlimit` is named completely differently than `register_on_userlimit_check`.

The same name should be used so that it's easier to find stuff, for example `register_can_bypass_userlimit` or something, similarly to `on_joinplayer`/`register_on_joinplayer`.

Reply:
i have some hesitation about consistency against textual representation, i'm neutral on the representation

Reply:
If you want to make an abstraction that involves different naming, it should be at some other place. ScriptApi is supposed to be a thin wrapper to Lua and changing names there is only confusing.

Reply:
yeah, i renamed too to can_bypass_userlimit :)